mango-core/src/main/scala/org/bdgenomics/mango/models/TrackedLayout.scala

```diff
@@ -57,6 +59,27 @@ object TrackedLayout {
     val ref2 = rec2._1
     ref1.overlaps(ref2)
   }
+
+  def overlaps[T](rec1: (ReferenceRegion, T), recs2: Seq[(ReferenceRegion, T)]): Boolean = {
+    val ref1 = rec1._1
+    val recs2List = recs2.toList
+    val start: Long = recs2List.min(Ordering.by((r: (ReferenceRegion, T)) => r._1.start))._1.start
+    val end: Long = recs2List.max(Ordering.by((r: (ReferenceRegion, T)) => r._1.end))._1.end
+    ref1.overlaps(new ReferenceRegion(recs2.last._1.referenceName, start, end))
```

Comment:
I think you could simplify the code:

```
val start: Long = recs2List.min(Ordering.by((r: (ReferenceRegion, T)) => r._1.start))._1.start
val end: Long = recs2List.max(Ordering.by((r: (ReferenceRegion, T)) => r._1.end))._1.end
new ReferenceRegion(recs2.last._1.referenceName, start, end)
```

To:

```
recs2.map(_._2).reduce((v1, v2) => v1.hull(v2))
```

The [hull](https://github.com/bigdatagenomics/adam/blob/master/adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala#L170) function computes the convex hull of two reference regions.